cpp/cmake_modules/FindGTest.cmake

```diff
@@ -38,34 +40,60 @@ elseif ( GTest_HOME )
     list( APPEND _gtest_roots ${GTest_HOME} )
 endif()
 
+set(GTEST_STATIC_LIB_NAME
+  ${CMAKE_STATIC_LIBRARY_PREFIX}gtest${CMAKE_STATIC_LIBRARY_SUFFIX})
+set(GTEST_MAIN_STATIC_LIB_NAME
+  ${CMAKE_STATIC_LIBRARY_PREFIX}gtest_main${CMAKE_STATIC_LIBRARY_SUFFIX})
+set(GTEST_SHARED_LIB_NAME
+  ${CMAKE_SHARED_LIBRARY_PREFIX}gtest${CMAKE_SHARED_LIBRARY_SUFFIX})
+set(GTEST_MAIN_SHARED_LIB_NAME
+  ${CMAKE_SHARED_LIBRARY_PREFIX}gtest_main${CMAKE_SHARED_LIBRARY_SUFFIX})
+
 # Try the parameterized roots, if they exist
-if ( _gtest_roots )
-    find_path( GTEST_INCLUDE_DIR NAMES gtest/gtest.h
-        PATHS ${_gtest_roots} NO_DEFAULT_PATH
-        PATH_SUFFIXES "include" )
-    find_library( GTEST_LIBRARIES NAMES gtest gtest_main
-        PATHS ${_gtest_roots} NO_DEFAULT_PATH
-        PATH_SUFFIXES "lib" )
-else ()
-    find_path( GTEST_INCLUDE_DIR NAMES gtest/gtest.h )
-    find_library( GTEST_LIBRARIES NAMES gtest )
-endif ()
+if(_gtest_roots)
```

Comment:
We should standardize on a CMake indentation rule, and ideally (but not sure it's possible) enforce it. Apparently there's [cmake_format](https://github.com/cheshirekow/cmake_format).

Reply:
@pitrou Can you make a JIRA for that? Then we can implement that.

Reply:
See https://issues.apache.org/jira/browse/ARROW-4363